Rules.md

```diff
@@ -178,67 +178,60 @@ Sea-sea movement is not possible for a unit if any land region at the strait is
 
 ## Battles
 
-When a move proceeds into an enemy region, a battle is initiated with all moved units.
+When a move proceeds into an enemy region, a battle is initiated between all units moved into the
+region versus the enemy units already in that region.
 
 ### Battle mechanic
 
-A battle has one or more rounds. Every round has following parts:
+A battle has the following phases:
 
 1. **Artillery strike:**
-    * Every healthy attacking artillery kills 1 unit (\*)
+    * Every attacking artillery kills 1 unit (\*).
 1. **Rolling dice:**
-    * The attacker gets a dice for every healthy attacking **infantry unit**
-    * The defender gets a dice for every healthy defending **unit** (infantry + artillery)
+    * The attacker gets a dice for every attacking **infantry unit**
+    * The defender gets a dice for every surviving defending **unit** (infantry + artillery)
     * Every attacker roll of
-        * **3-4** wounds an enemy unit, (\*)
-        * **5-6** kills an enemy unit. (\*)
+        * **1-2** equals 1 military power,
+        * **3-4** equals 2 military power,
+        * **5-6** equals 2 military power and kills an enemy unit. (\*)
     * Every defender roll of
-        * **1-4** wounds an enemy unit, (\*)
-        * **5-6** kills an enemy unit. (\*)
-
-    Kills are resolved before wounds.
-1. **The attacker has no healthy units left or decides to retreat:** All units move back to the region
-   where the move started.
-
-   Edge case: If the attacker has no healthy units left and the defender has no units left, the
-   attacker has to retreat. Any command tokens on the attacked region are returned to the defender's
-   supply.
-1. **The defender has no healthy units left or decides to retreat:** All units (including wounded units)
-   move to a single adjacent land region that is either unoccupied or owned by the defender.
+        * **1-4** equals 2 military power,
+        * **5-6** equals 2 military power and kills an enemy unit. (\*)
+   The attacker gets an additional 2 military power for each of his surviving artillery.
```

Comment:
I know, my goal was to keep it simple but still as exact as possible to the original probabilities.
This was needed for that.

I'm always open for simplifications that do have an impact on the probabilities.